spec/deaf_grandma_spec.rb

```diff
@@ -7,12 +7,17 @@
   it "says 'SPEAK UP SONNY!' when we speak regularly" do
     expect(script.speak("Hi Grandma")).to eq "SPEAK UP SONNY!"
   end
-  
+
   it "says 'NOT SINCE 1964!' when we yell" do
     #implement your test here
+    expect(script.speak("HI")).to eq "NOT SINCE 1964!"
   end
 
   it "EXTRA CREDIT: How would you test yelling BYE?" do
     #implement your test here
+    expect(script.speak("BYE")).to eq "DON'T LEAVE ME"
+    expect(script.speak("BYE")).to eq "DON'T LEAVE ME"
+    expect(script.speak("BYE")).to eq "DON'T LEAVE ME"
+    expect(script.speak("BYE")).to eq "SEE YOU LATER SONNY!"
```

Comment:
Yeah I see what you mean. But when I run the tests, I get it all passing.
Weird. I'm still trying to understand RSpec.

I'll fix it and do another PR.

On Sat, Mar 5, 2016 at 7:29 PM, Joe Reich <notifications@github.com> wrote:

> In spec/deaf_grandma_spec.rb
> <https://github.com/paircolumbus/MoreDrillsWithRSpec/pull/43#discussion_r55130652>
> :
>
> >    end
> >
> >    it "EXTRA CREDIT: How would you test yelling BYE?" do
> >      #implement your test here
> > +    expect(script.speak("BYE")).to eq "DON'T LEAVE ME"
> > +    expect(script.speak("BYE")).to eq "DON'T LEAVE ME"
> > +    expect(script.speak("BYE")).to eq "DON'T LEAVE ME"
> > +    expect(script.speak("BYE")).to eq "SEE YOU LATER SONNY!"
>
> are these tests passing? Looks like the count is off.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/paircolumbus/MoreDrillsWithRSpec/pull/43/files#r55130652>
> .
>